docs/server/source/dev-and-test/setup-run-node.md

```diff
@@ -147,22 +147,16 @@ docker-compose run --rm bdb py.test -v --database-backend=mongodb
 
 ### Accessing the HTTP API
 
-A quick check to make sure that the BigchainDB server API is operational:
+You can do quick check to make sure that the BigchainDB server API is operational:
 
 ```bash
 curl $(docker-compose port bdb 9984)
 ```
 
-should give you something like:
+The result should be a JSON object (inside braces like { })
```

Comment:
The part in brackets is redundant IMO. If you want to be more specific, you can say `The result should be a JSON dictionary`.